app/src/androidTest/java/com/github/se/signify/end2end/HomeScreenFeaturesEnd2endTest.kt

```diff
@@ -103,6 +103,11 @@ class HomeScreenFeaturesEnd2endTest {
     composeTestRule.onNodeWithTag("HomeScreen").assertIsDisplayed()
     composeTestRule.onNodeWithTag("QuizButton").performClick()
     composeTestRule.onNodeWithTag("QuizTitle").assertIsDisplayed()
+
+    // Simulate choosing the correct solution based on mocked data
```

Comment:
Since quizzes are not tracked and are considered a free game in a sense, it’s a good idea to test them while interacting with the home screen.